packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -400,8 +400,14 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{
       }
 
       if (customConfig.fillBelowTo) {
-        const fillBelowToField = frame.fields.find((f) => f.name === customConfig.fillBelowTo);
-        const fillBelowDispName = getFieldDisplayName(fillBelowToField!, frame, allFrames);
+        const fillBelowToField = frame.fields.find((field) =>
+          [field.name, field.config?.displayName, field.config?.displayNameFromDS, field.state?.displayName].includes(
```

Comment:
rather than try to enumerate them... maybe just use the function?  (it will most likely grab from cached state anyway)

somethign like
```
let fillBelowDispName = customConfig.fillBelowTo;
const fillBelowToField = frame.fields.find((f) => {
  return (fillBelowDispName === f.name || fillBelowDispName === getFieldDisplayName(f, frame, allFrames));
})
```

Reply:
The reason I enumerated was because I needed to find a field name possibly without overrides. The options in the `fillBelowTo` are without field overrides (e.g: display name). Searching for fields with the result of `getFieldDisplayName` would result in post override names, which wouldn't match the `fill bellow to` option, in some scenarios. 

E.g: I have a display name override for a field that is also set as a `fill bellow to` field. So my `customConfig.fillBelowTo` value is the name of the field before the display name override, and that's the one I need to find. Sometimes `field.name` will do, other times it might be `field.config.displayNameFromDS`. But `getFieldDisplayName` func will always return the name of the field with the display name override applied, which doesn't help.

Reply:
I changed the code to use the function if other checks fail